src/book/03-guides/01-main-codebase/04-maintaining-subsystems.mdx

```diff
@@ -69,21 +71,21 @@ An automatic camera calibration tool is available in the NUbots repository. See
 
 After updating the Visual Mesh in the NUbots repository, it should be tested before merging. Refer to the [Getting Started guide](/guides/main/getting-started) for assistance for the following steps.
 
-1. Build the code, ensuring `ROLE_visualmesh` is set to `ON` in `./b configure -i`, and install it to the robot. Ensure the new configuration file is installed by using the `-cu` or `-co` options when installing - check out the [Build System page](/system/foundations/build-system#install) to find out more about options when installing onto the robot.
+1. Build the code, ensuring `ROLE_test-visualmesh` is set to `ON` in `./b configure -i`, and install it to the robot. Ensure the new configuration file is installed by using the `-cu` or `-co` options when installing - check out the [Build System page](/system/foundations/build-system#install) to find out more about options when installing onto the robot.
 
 2. When your new Visual Mesh is installed onto the robot, connect to the robot using:
 
    ```bash
    ssh nubots@<address>
    ```
 
-3. Ensure NUsight is on:
+3. Ensure the robot is sending vision data:
 
    ```bash
    nano config/NetworkForwarder.yaml
    ```
 
-   Turn `vision object` and `compressed images` on. Run NUsight using `yarn prod` and navigate to the NUsight page in your browser. More on NUsight can be found on [the NUsight NUbook page](/system/tools/nusight).
+   CompressedImage, Balls, Goals and GreenHorizon should be on. Run NUsight using `yarn prod` and navigate to the NUsight page in your browser. More on NUsight can be found on [the NUsight NUbook page](/system/tools/nusight).
```

Comment:
Might be worth mentioning that NUsight has to have been built already if using `yarn prod` - need to run `yarn build` if it hasn't yet

Reply:
```suggestion
   CompressedImage, Balls, Goals and GreenHorizon should be on. Run NUsight using `yarn prod` and navigate to the NUsight page in your browser. More on NUsight can be found on [the NUsight NUbook page](/system/tools/nusight). If you have not already set up and built NUsight, refer to the [Getting Started page](/guides/main/getting-started).
```